src/main/resources/consumer.properties

```diff
@@ -15,7 +15,6 @@
 # Intentionally not respecting 'group.id' setting here as I enforce a unique group id to avoid any potential group sharing whch might inadvertantly cause a message being routed to another consumer. This way I guarantee we should receive back our unique generated message as part of the test
 
 bootstrap.servers = localhost:9092
-security.protocol = PLAINTEXT
```

Comment:
ditto with the jaas config changes I suspect there will need to be different properties configs for kerberos and non-kerberos